src/components/Assets/Assets.tsx

```diff
@@ -106,7 +132,7 @@ export default function Assets(props: Props) {
         </div>
       </div>
     )
-  }, [isMobile, nftFilters.category, view, profileName])
+  }, [isMobile, hasAdditionalFiltersEnabled, nftFilters.category, view, profileName])
```

Comment:
`handleClearFilters` seems to be required here:
```suggestion
  }, [isMobile, hasAdditionalFiltersEnabled, nftFilters.category, view, profileName, handleClearFilters])
```